dashboard/src/features/shared/actions/form.js

```diff
@@ -15,14 +16,10 @@ export default function(type, options = {}) {
     created,
     submitForm: (data) => {
       const className = options.className || type.charAt(0).toUpperCase() + type.slice(1)
+      let promise = new Promise((resolve) => resolve())
```

Comment:
We can't remove the temporary because we want to optionally chain the Xpub calls and ensure they run before the object creation.